### PR TITLE
Update setup.md

### PR DESCRIPTION
**Pull request by johnisanerd**
Updating to encourage the user to build a virtual environment first.

### PR Quality Checklist
- [ ] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -194,6 +194,14 @@ docker run -it --env-file=.env -v $PWD:/app --rm auto-gpt --gpt3only --continuou
 
 ### Run without Docker
 
+#### Create a Virtual Environment
+Create a virtual environment to run in.  
+        :::shell
+        python -m venv venvAutoGPT
+        source venvAutoGPT/bin/activate
+        pip3 install --upgrade pip
```
Comment (Pwuts) [inline docs/setup.md]: The formatting of this is broken, see the preview

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

File: docs/setup.md (hunk 1 above)
Comment (lc0rp) [reply docs/setup.md]: @johnisanerd we'd like to get this into 0.3.2. Could you resolve this comment and update the branch?

Comment (johnisanerd) [reply docs/setup.md]: Taking a look right now, thanks for the reminder!

**Review by richbeales — APPROVED** [review]

**Pull request merged**